crates/sui-types/src/storage.rs

```diff
@@ -671,7 +671,7 @@ impl From<&ObjectRef> for ObjectKey {
 /// Fetch the `ObjectKey`s (IDs and versions) for non-shared input objects.  Includes owned,
 /// and immutable objects as well as the gas objects, but not move packages or shared objects.
 pub fn transaction_input_object_keys(tx: &SenderSignedData) -> SuiResult<Vec<ObjectKey>> {
-    use crate::messages::InputObjectKind::{ImmOrOwnedMoveObject, MovePackage, SharedMoveObject};
+    use crate::messages::InputObjectKind::*;
```

Comment:
Channeling @tnowacki, I would say, there's a small risk of accidentally binding an identifier pattern, rather than specifying a constructor to match against when `use`-ing enumerands in this way (not in this case so much, but generally), and it's better to alias them instead as a rule:

```suggestion
    use crate::messages::InputObjectKind as I;
```